### PR TITLE
Add HotS Replay Uploader 2.0.0

### PR DESCRIPTION
**Pull request by thorstenschaefer**
This is the new cask for the HotS Replay Uploader as suggested [here](https://github.com/eivindveg/HotSUploader/issues/39).

### CODE REVIEW

File: Casks/hots-replay-uploader.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'hots-replay-uploader' do
+  version '2.0.0'
+  sha256 'c0d5775c47b2c57a2b63bddbbb7f4c380874cb37c277b4477491fe4d7be875e2'
+  url 'https://github.com/eivindveg/HotSUploader/releases/download/v2.0.0/HotS.Replay.Uploader-2.0.0.dmg'
```
Comment (adityadalal924) [inline Casks/hots-replay-uploader.rb]: Please use `#{version}` instead of hardcoding in an app version

File: Casks/hots-replay-uploader.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'hots-replay-uploader' do
+  version '2.0.0'
+  sha256 'c0d5775c47b2c57a2b63bddbbb7f4c380874cb37c277b4477491fe4d7be875e2'
```
Comment (adityadalal924) [inline Casks/hots-replay-uploader.rb]: Add an empty line after the sha256 stanza

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in d4e0c4b9b4dc5cb71f523bfac1c00e117e7e00aa. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by thorstenschaefer** [comment]
Thanks @adityadalal924 and sorry for not getting back more quickly, I just came back home. Everything makes sense to me, but I'd like to suggest a change in the Contributions documentation (https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md): The examples section did not use #{version} and typically serves as a blue print for new cask creators like me. Maybe it should be added there too in order to avoid mistakes like mine?

**Comment by adityadalal924** [comment]
Excellent, thanks for the feedback. Proposed some documentation changes in https://github.com/caskroom/homebrew-cask/pull/15564 that should hopefully make things clearer.

**Pull request closed without merging**